crates/astria-bridge-withdrawer/local.env.example

```diff
@@ -48,6 +48,9 @@ ASTRIA_BRIDGE_WITHDRAWER_ROLLUP_ASSET_DENOMINATION="nria"
 # Should match the bridge address in the geth rollup's bridge configuration for that asset.
 ASTRIA_BRIDGE_WITHDRAWER_SEQUENCER_BRIDGE_ADDRESS=""
 
+# The timeout duration for `Ics20Withdrawal`s in seconds.
+ASTRIA_BRIDGE_WITHDRAWER_ICS20_WITHDRAWAL_TIMEOUT_DURATION=300
```

Comment:
Might make the units more visible if we named this `ASTRIA_BRIDGE_WITHDRAWER_ICS20_WITHDRAWAL_TIMEOUT_SECONDS`?